.gitignore

```diff
@@ -11,7 +11,8 @@
 
 # compiled core library output
 /libs/core/loader
-/libs/core/dist
+/libs/core/dist/
+/libs/core/scss/
```

Comment:
In the publish script, we are copying them, not moving them. We do this because we want the scss to be located on the top level when installed in node_modules so it can be referenced with `@kirbydesign/core`. 

The reason that `/libs/core/loader`, `/libs/core/dist/` and `/libs/core/scss/` rules exist is that we need to be able to reference these foldes from package.json, so that npm packs them properly, under the `dist` directory. I have tried to move all these under a single dist folder, but unfortunately the stencil compiler is very strict on the format of the package.json paths to the dist files.